tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -11882,6 +11913,33 @@ bool hlsl::ShouldSkipNRVO(clang::Sema &sema, clang::QualType returnType,
   return false;
 }
 
+bool hlsl::HasLongVecs(const QualType &qt) {
+  if (qt.isNull()) {
+    return false;
+  }
```

Comment:
nit: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements
```suggestion
  if (qt.isNull())
    return false;
```

Reply:
Not part of this PR. I've responded to it here: https://github.com/microsoft/DirectXShaderCompiler/pull/7143/commits/50d1af5b645651b7ee4d4ef063bdc88c7d6790d4